FCCee/compact/FCCee_o2_v04/FCCee_o1_v04.xml

```diff
@@ -0,0 +1,372 @@
+        <constant name="tracker_region_rmax" value="OuterTracker_outer_radius"/>
+
+        <constant name="GlobalTrackerReadoutID" type="string" value="system:5,side:-2,layer:6,module:11,sensor:8"/>
+    </define>
+    
+    <limits>
+        <limitset name="cal_limits">
+            <limit name="step_length_max" particles="*" value="5.0" unit="mm" />
+        </limitset>
+    </limits>
+    <regions>
+        <region name="BeampipeRegion"            />
+        <region name="VertexBarrelRegion"        />
+        <region name="VertexEndcapRegion"        />
+        <region name="InnerTrackerBarrelRegion"  />
+        <region name="OuterTrackerBarrelRegion"  />
+        <region name="InnerTrackerEndcapRegion"  />
+        <region name="OuterTrackerEndcapRegion"  />
+    </regions>
+
+
+    <display>
+      <vis name="VXDVis"        alpha="0.1" r="0.1" 	g=".5"      b=".5"    showDaughters="true"  visible="false"/>
+      <vis name="ITVis"       	alpha="1.0" r="0.54"  	g="0.43"    b="0.04"  showDaughters="true"  visible="true"/>
+      <vis name="OTVis"       	alpha="1.0" r="0.8"   	g="0.8"     b="0.4"   showDaughters="true"  visible="false"/>
+      <vis name="ECALVis"     	alpha="1.0" r="0.0"   	g="0.48"    b="0.0"   showDaughters="true"  visible="true"/>
+      <vis name="HCALVis"     	alpha="1.0" r="0.74" 	g="0.81"    b="0.55"  showDaughters="true"  visible="true"/>
+      <vis name="SOLVis"      	alpha="1.0" r="0.4"   	g="0.4"     b="0.4"   showDaughters="true"  visible="true"/>
+      <vis name="YOKEVis"     	alpha="1.0" r="0.0"   	g="0.56"    b="0.28"  showDaughters="true"  visible="true"/>
+      <vis name="LCALInstrVis"  alpha="1.0" r="0.35"  	g="0.0"     b="0.47"  showDaughters="true"  visible="true"/>
+      <vis name="LCALVis"    	alpha="1.0" r="0.25"  	g="0.88"    b="0.81"  showDaughters="true"  visible="true"/>
+      <vis name="LCALCoolVis"   alpha="1.0" r="0.2"   	g="0.6"     b="0"     showDaughters="true"  visible="true"/>
+      <vis name="CompSolVis"    alpha="1.0" r="0.5"   	g="0.5"     b="0.0"   showDaughters="true"  visible="true"/>
+      <vis name="ScreenSolVis"  alpha="1.0" r="1"   	g="1"       b="0"     showDaughters="true"  visible="true"/>
+      <vis name="TantalumVis"   alpha="1.0" r="1"   	g="0.5"     b="0.5"   showDaughters="true"  visible="true"/>
+      <vis name="SupportVis"  	alpha="1"   r="0.2"   	g="0.2"     b="0.2"   showDaughters="true" visible="true"/>
+    </display>
+
+    <include ref="${DD4hepINSTALL}/DDDetectors/compact/detector_types.xml"/>
+
+    <include ref="Beampipe_o4_v04_noNotch_W_n02.xml"/>
+    <include ref="HOMAbsorber_v00.xml"/>
+    <include ref="BeamInstrumentation_o3_v02_fitShield.xml"/>
+
+    <include ref="LumiCal_o3_v02_03.xml"/>
+
+    <include ref="Vertex_o4_v05.xml"/>
+    <include ref="InnerTracker_o2_v06_02.xml"/>
+    <include ref="OuterTracker_o2_v06_02.xml"/>
+
+    <include ref="ECalBarrel_o2_v01_03.xml"/>
```

Comment:
Here you need to include `ECalLArBarrel_o2_v01_01.xml` and remove `ECalBarrel_o2_v01_03.xml`